src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/Reindex/ReindexJobTaskTests.cs

```diff
@@ -224,10 +230,12 @@ public async Task GivenRunningJob_WhenExecuted_ThenQueuedQueryCompleted()
             await _reindexJobTask.ExecuteAsync(job, _weakETag, _cancellationToken);
 
             Assert.Equal(OperationStatus.Completed, job.Status);
-            Assert.Equal(10, job.Count);
+            Assert.Equal(_mockedSearchCount * 2, job.Count);
             Assert.Equal(4, job.Progress);
-            Assert.Equal("Appointment,AppointmentResponse", job.ResourceList);
-            Assert.Equal("http://hl7.org/fhir/SearchParameter/AppointmentResponse-appointment", job.SearchParamList);
+            Assert.Contains("Appointment", job.ResourceList);
+            Assert.Contains("AppointmentResponse", job.ResourceList);
+            Assert.Contains("http://hl7.org/fhir/SearchParameter/AppointmentResponse-appointment", job.SearchParamList);
+            Assert.Contains("http://hl7.org/fhir/SearchParameter/Appointment-date", job.SearchParamList);
             Assert.Collection<ReindexJobQueryStatus>(
```

Comment:
>Collection [](start = 19, length = 10)

I think tests failing on order of ResourceTypes

Reply:
Thanks, I removed the ordering in c36800e.